integration-test/oauth-tooling/tokenCache.spec.ts

```diff
@@ -345,9 +345,9 @@ describe('tokenCache', () => {
 
     let promise = tokenService.refreshAllTokens()
       .then((res) => {
-        return res.map(tokeninfo => tokeninfo['access_token']);
+        const tokens = res.map(tokeninfo => tokeninfo['access_token']);
+        return tokens.sort();
```

Comment:
The test fails randomly as the order of the resulting array created from an unsorted, async created object is not guaranteed. Sort enforces the order that can be matched to the expected result.